src/TypedEventEmitter.ts

```diff
@@ -1,26 +1,17 @@
 /**
  * An EventEmitter that supports events given by the interface in {T}
  * Where each Key represents the eventName and the type represents the payload
- * This interface is based on the implementation here: https://github.com/Gozala/events/blob/master/events.js
  * 
  * @export
  * @interface TypedEventEmitter
  * @template T
  */
 export default interface TypedEventEmitter<T> {
-    on<K extends keyof T>(name: K, listener: (v: T[K]) => void): this;
     addListener<K extends keyof T>(event: K, listener: (v: T[K]) => void): this;
```

Comment:
I think where appropriate we can use the new interface.  So here, we could probably return the `EmitterSubscription`.  It is a breaking change, but aligns with the NativeEventEmitter implementation better

Reply:
Done https://github.com/cjam/react-native-spotify-remote/pull/189/commits/b6abe5bfe8418da7fd13c6edbdd91ebe766ff18c